pkg/jx/cmd/controller_build.go

```diff
@@ -101,6 +101,12 @@ func (o *ControllerBuildOptions) Run() error {
 	o.EnvironmentCache = kube.CreateEnvironmentCache(jxClient, ns)
 
 	if o.InitGitCredentials {
+		// lets validate we have git configured
+		_, _, err = gits.EnsureUserAndEmailSetup(o.Git())
+		if err != nil {
+		  return err
```

Comment:
I think would be nice to wrap this error with a message.

Reply:
all the error conditions are already wrapped inside the `EnsureUserAndEmailSetup()` function so didn't see any need to wrap again